nuxeo/templates/configmap.yaml

```diff
@@ -92,9 +92,17 @@ data:
     {{- end }}
 {{- end }}
     metrics.enabled={{ .Values.metrics.enabled }}
-{{- if .Values.metrics.stackDriver.enabled }}
-    metrics.stackdriver.enabled={{ .Values.metrics.stackDriver.enabled }}
+    metrics.streams.enabled={{ .Values.metrics.streams.enabled }}
```

Comment:
this should be moved inside the if metrics.enabled block, this is required for all metrics/tracing reporters